### PR TITLE
docs: add hermione -> testplane migration guide

### PR DESCRIPTION
**Pull request by KuznetsovRoman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,9 +86,11 @@ The documentation is divided into several sections:
 - [Testplane Events](docs/events.md)
 - [Testplane Programmatic API](docs/programmatic-api.md)
 
-## Rename from "hermione"
+## Rename from "Hermione"
 
-This project was formerly known as "hermione", but eventually some copyright and trademark issues surfaced, leading to the decision to rebrand. After some discussion, we settled on "Testplane" as the official new title. We've tailored Testplane `v1` to be a drop-in replacement for Hermione `v8`.
+This project was formerly known as "Hermione", but eventually some copyright and trademark issues surfaced, leading to the decision to rebrand. After some discussion, we settled on "Testplane" as the official new title. We've tailored Testplane `v1` to be a drop-in replacement for Hermione `v8`.
```
Comment (KuznetsovRoman) [inline README.md:91]: Replaced "hermione" to "Hermione", where it is necessary

File: docs/migration-from-hermione.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+So, to migrate from Hermione to Testplane you need to:
+- Uninstall hermione (`npm uninstall hermione`);
+- Install testplane (`npm install -D testplane`);
+- Replace all `import ... from "hermione"` and `require("hermione")` to `import ... from "testplane"` and `require("testplane")`;
+- Replace `declare module "hermione"` to `declare module "testplane"`;
+- Replace "hermione" to "testplane" in `compilerOptions.types` of your `tsconfig.json` files.
+
+Other than that, everything should be fine. Hermione plugins are fully compatible with Testplane@1.
+
+These things only work in Testplane@1 and will be removed in Testplane@2:
```
Comment (sipayRT) [inline docs/migration-from-hermione.md:14]: I'm not sure we need to talk about it right now

File: docs/migration-from-hermione.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+## Migration from Hermione
+
+Testplane@1 is backward compatible with Hermione@8.
+
+So, to migrate from Hermione to Testplane you need to:
```
Comment (sipayRT) [inline docs/migration-from-hermione.md:5]: you can also add information that for a seamless move in CI, we provide both `testplane` and `hermione` binaries

File: docs/migration-from-hermione.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+- Uninstall hermione (`npm uninstall hermione`);
+- Install testplane (`npm install -D testplane`);
+- Replace all `import ... from "hermione"` and `require("hermione")` to `import ... from "testplane"` and `require("testplane")`;
+- Replace `declare module "hermione"` to `declare module "testplane"`;
```
Comment (sipayRT) [inline docs/migration-from-hermione.md:9]: you can merge these two points as "Replace all imports, requires and declarations [examples here]"

File: docs/migration-from-hermione.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+## Migration from Hermione
+
+Testplane@1 is backward compatible with Hermione@8.
+
+So, to migrate from Hermione to Testplane you need to:
+- Uninstall hermione (`npm uninstall hermione`);
+- Install testplane (`npm install -D testplane`);
```
Comment (sipayRT) [inline docs/migration-from-hermione.md:7]: "Replace `hermione` deps with `tesplane` in package.json - `npm uninstall hermione && npm install -D testplane`"

**Review by sipayRT — APPROVED** [review]
the main idea is to reduce the number of points so that it does not seem that moving can be difficult

File: docs/migration-from-hermione.md (hunk 2 above)
Comment (KuznetsovRoman) [reply docs/migration-from-hermione.md:14]: I wrote it to show "you dont need to change any of these in Testplane@1"

File: docs/migration-from-hermione.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
+## Migration from Hermione
+
+Testplane@1 is backward compatible with Hermione@8.
+
+So, to migrate from Hermione to Testplane you need to:
+- Replace hermione deps with tesplane in package.json (`npm uninstall hermione && npm install -D testplane`);
+- Replace "hermione" to "testplane" in `compilerOptions.types` of your `tsconfig.json` files;
```
Comment (shadowusr) [inline docs/migration-from-hermione.md:7]: Small typo: Replace with or rename to.

```
Replace "hermione" with "testplane" in the `compilerOptions.types` field of your `tsconfig.json` file;
```

File: docs/migration-from-hermione.md (hunk 3 above)
Comment (shadowusr) [reply docs/migration-from-hermione.md:5]: I'd omit "So":

To migrate from Hermione to Testplane you'll need to:

Not critical to me.

File: README.md [hunk 7]
```diff
@@ -86,9 +86,11 @@ The documentation is divided into several sections:
...
+
+You can read about migration in details in section [Migration from Hermione](docs/migration-from-hermione.md).
```
Comment (shadowusr) [inline README.md:93]: I don't quite like two in's, but feel free to keep it if you wish, not critical to me

```
Learn more about about migration from Hermione to Testplane in [the Docs](docs/migration-from-hermione.md).
```

**Review by shadowusr — APPROVED** [review]
Great guide overall, like it! Left a couple of minor comments regarding a few typos and other small issues.

**Comment by shadowusr** [comment]
ah, another thing. Did you try running `npm run toc`? This command is used to generate tables of contents in docs files. If you don't need one in your newly created file, include `<!-- DOCTOC SKIP -->` to the beginning of the file.

**Pull request merged**